content/pt-br/docs/reference/access-authn-authz/authorization.md

```diff
@@ -0,0 +1,250 @@
+ * **API** - Indica se a solicitação é para um recurso de API.
+ * **Request path** - Caminho para diversos endpoints sem recursos, como `/api` ou `/healthz`.
+ * **API request verb** - Verbos da API como `get`, `list`, `create`, `update`, `patch`, `watch`, `delete` e `deletecollection` que são utilizados para solicitações de recursos. Para determinar o verbo de requisição para um endpoint de recurso de API , consulte [Determine the request verb](/docs/reference/access-authn-authz/authorization/#determine-the-request-verb).
+ * **HTTP request verb** - Métodos HTTP em letras minúsculas como `get`, `post`, `put` e `delete` que são utilizados para requisições que não são de recursos.
+ * **Resource** - O identificador ou nome do recurso que está sendo acessado (somente para requisições de recursos) -- Para requisições de recursos usando os verbos `get`, `update`, `patch` e `delete`, deve-se fornecer o nome do recurso.
+ * **Subresource** - O sub-recurso que está sendo acessado (somente para solicitações de recursos).
+ * **Namespace** - O namespace do objeto que está sendo acessado (somente para solicitações de recursos com namespace).
+ * **API group** - O {{< glossary_tooltip text="API Group" term_id="api-group" >}} sendo acessado (somente para requisições de recursos). Uma string vazia designa o _core_ [API group](/docs/reference/using-api/#api-groups).
+
+## Determine o verbo da requisição
+
+**Requisições de não-recursos**
+
+Requisições para endpoints diferentes de `/api/v1/...` ou `/apis/<group>/<version>/...`
+são considerados "requisições sem recursos" e usam o método HTTP em letras minúsculas da solicitação como o verbo.
+Por exemplo, uma solicitação `GET` para endpoints como `/api` ou `/healthz` usaria `get` como o verbo.
+
+**Requisições de recursos**
+Para determinar o verbo de requisição para um endpoint de API de recurso, revise o verbo HTTP
+utilizado e se a requisição atua ou não em um recurso individual ou em uma
+coleção de recursos:
+
+HTTP verb | request verb
+----------|---------------
+POST      | create
+GET, HEAD | get (para recursos individuais), list (para coleções, includindo o conteúdo do objeto inteiro), watch (para assistir um recurso individual ou coleção de recursos)
+PUT       | update
+PATCH     | patch
+DELETE    | delete (para recursos individuais), deletecollection (para coleções)
+
+Às vezes, o Kubernetes verifica a autorização para permissões adicionais utilizando verbos especializados. Por exemplo:
+
+* [PodSecurityPolicy](/docs/concepts/security/pod-security-policy/)
+  * `use` verbo em recursos `podsecuritypolicies` no grupo `policy` de API.
+* [RBAC](/docs/reference/access-authn-authz/rbac/#privilege-escalation-prevention-and-bootstrapping)
+  * `bind` e `escalate` verbos em `roles` e recursos `clusterroles` no grupo `rbac.authorization.k8s.io` de API.
+* [Authentication](/docs/reference/access-authn-authz/authentication/)
```

Comment:
It can be replaced with [pt-br](https://kubernetes.io/pt-br/docs/reference/access-authn-authz/authentication/) version.
```suggestion
* [Authentication](/pt-br/docs/reference/access-authn-authz/authentication/)
```